frontend/components/layout/Sidebar.tsx

```diff
@@ -53,33 +53,41 @@ const Sidebar = () => {
 
   const { organisations, activeOrganisation } = useContext(organisationContext)
 
-  const showOrgsMenu = organisations === null ? false : organisations?.length > 1
+  const showOrgsMenu = organisations && organisations.length > 1
 
   const isOwner = organisations?.some((org) => org.role?.toLowerCase() === 'owner')
 
   const OrgsMenu = () => {
+    const OrgLabel = () => (
+      <div className="p-2 text-neutral-500 flex items-center justify-between w-full bg-neutral-500/10 ring-1 ring-inset ring-neutral-400/10 rounded-lg">
+        <div className="flex flex-col gap-0.5 min-w-0">
+          <div>
+            <PlanLabel plan={activeOrganisation?.plan!} />
+          </div>
+          <span className="truncate font-semibold tracking-wider text-lg">
+            {activeOrganisation?.name}
+          </span>
+        </div>
+      </div>
+    )
+
+    if (!showOrgsMenu) return <OrgLabel />
+
     return (
       <Menu as="div" className="relative inline-block text-left w-full">
         {({ open }) => (
           <>
-            <Menu.Button
-              as="div"
-              className="p-2 text-neutral-500 cursor-pointer flex items-center justify-between w-full group bg-neutral-500/10 ring-1 ring-inset ring-neutral-400/10 rounded-lg"
-            >
-              <div className="flex flex-col gap-0.5 min-w-0">
-                <div>
-                  <PlanLabel plan={activeOrganisation?.plan!} />
-                </div>
-                <span className="truncate font-semibold tracking-wider text-lg">
-                  {activeOrganisation?.name}{' '}
-                </span>
+            <Menu.Button className="w-full">
+              <div className="relative">
+                <OrgLabel />
+                <FaChevronDown
+                  className={clsx(
+                    'absolute right-2 top-1/2 -translate-y-1/2 transition-transform',
```

Comment:
Not a huge fan of this absolute positioning, but its fine for now. We can improve it later. Ideally we would just use `flex items-center` to handle this positioning.